flutter/shell/platform/tizen/flutter_tizen.cc

```diff
@@ -240,6 +240,11 @@ void* FlutterDesktopViewGetNativeHandle(FlutterDesktopViewRef view_ref) {
   return view->tizen_view()->GetNativeHandle();
 }
 
+uint32_t FlutterDesktopViewGetResourceId(FlutterDesktopViewRef view_ref) {
+  flutter::FlutterTizenView* view = ViewFromHandle(view_ref);
+  return view->tizen_view()->GetResourceId();
```

Comment:
This new API just for video_palyer_avplay plugin(https://github.com/xiaowei-guan/plugins/commit/f3cff83de6d0ffbc198ea1fc1230dd7ac91165bc)
Actually we can extend this api for Ecore_Evas:
```cpp
   Ecore_Wl2_Window* wl_window = ecore_evas_wayland2_window_get(ee);
```
I only implemented it for Ecore_wayland, because this third-party wanted to apply the new API as soon as possible, I will add the rest of the code later.